crates/ruff_cli/src/commands/run.rs

```diff
@@ -226,3 +226,85 @@ with the relevant file contents, the `pyproject.toml` settings, and the followin
         }
     }
 }
+
+#[cfg(test)]
+#[cfg(unix)] // If you need to add a second test, move this cfg
```

Comment:
I might just suggest removing this comment -- it was confusing enough for me that I had to ask about it :joy:

Reply:
fair